cluster/service/cluster_service.go

```diff
@@ -45,28 +44,29 @@ func NewClusterService(context servicectx.ServiceContext, loader ConfigLoader) s
 }
 
 // CreateOrSaveClusterFromConfig creates clusters or save updated cluster info from config
-func (c clusterService) CreateOrSaveClusterFromConfig(ctx context.Context) error {
+func (s clusterService) CreateOrSaveClusterFromConfig(ctx context.Context) error {
```

Comment:
How about `cs` short name?

Reply:
I think that `s` for `service` is enough, and we've used a single letter for all other receivers, so I'd rather keep it as-is